generators/messaging-adapter-generator/src/main/java/uk/gov/justice/raml/jms/core/JmsEndpointGenerator.java

```diff
@@ -79,11 +79,11 @@ public void run(final Raml raml, final GeneratorConfig configuration) {
     therefore we need all messages with consecutive numbers there. Messages need to be in correct order in order to update the view correctly.
     */
 
-    private boolean shouldGenerateEventFilter(final Resource resource, final MessagingAdapterBaseUri baseUri) {
+    static boolean shouldGenerateEventFilter(final Resource resource, final MessagingAdapterBaseUri baseUri) {
```

Comment:
If we are making these two methods static and accessible from another class, then shouldn't they be in another Class?

Reply:
Moved to a separate class JmsEndPointGeneratorUtil